packages/ag-charts-types/src/chart/labelOptions.ts

```diff
@@ -15,7 +15,7 @@ export interface AgChartLabelOptions<TDatum, TParams> extends Toggleable, FontOp
     formatter?: Formatter<AgChartLabelFormatterParams<TDatum> & TParams>;
 }
 
-export interface AgChartLabelFormatterParams<TDatum> extends AgChartCallbackParams<TDatum> {
+export interface AgChartLabelFormatterParams<TDatum = any> extends AgChartCallbackParams<TDatum> {
```

Comment:
Is this needed?

Reply:
As above, to align with the grid convention for callbacks that can optionally be untyped

Reply:
Reverted